pyuvdata/uvcal/uvcal.py

```diff
@@ -1071,6 +1175,320 @@ def use_current_array_shapes(self):
         if self.freq_range is not None:
             self.freq_range = self.freq_range[0, :].tolist()
 
+    def remove_flex_jones(self, *, combine_spws=True):
+        """
+        Convert a flex-Jones UVData object into one with a standard Jones axis.
```

Comment:
UVData -> UVCal

I found several problems like this but I may have missed some. Maybe search in the file to find any more.

Reply:
Ah, good catch -- yes, I'll go through and search.

Reply:
Fixed in 766e32646768181b70ddd8d2e576d90ca57c254c.